auto_submit/lib/service/pull_request_validation_service.dart

```diff
@@ -59,11 +59,14 @@ class PullRequestValidationService extends ValidationService {
     final github.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
     final int prNumber = messagePullRequest.number!;
 
+    // If a pull request is currently in the merge queue do not touch it. Let
+    // the merge queue merge it, or kick it out of the merge queue.
     if (messagePullRequest.isMergeQueueEnabled) {
       if (result.repository!.pullRequest!.isInMergeQueue) {
         log.info(
           '${slug.fullName}/$prNumber is already in the merge queue. Skipping.',
         );
+        await pubsub.acknowledge(config.pubsubRevertRequestSubscription, ackId);
```

Comment:
I missed this last time. I haven't seen a real issue from this, but the pubsub message was successfully processed at this point, so acknowledging it makes sense.